src/stan/lang/generator/expression_visgen.hpp

```diff
@@ -163,23 +166,23 @@ namespace stan {
            << '('
            << fx.system_function_name_
            << "_functor__(), ";
-        generate_expression(fx.y0_, o_);
+        generate_expression(fx.y0_, NOT_USER_FACING, o_);
         o_ << ", ";
-        generate_expression(fx.t0_, o_);
+        generate_expression(fx.t0_, NOT_USER_FACING, o_);
         o_ << ", ";
-        generate_expression(fx.ts_, o_);
+        generate_expression(fx.ts_, NOT_USER_FACING, o_);
         o_ << ", ";
-        generate_expression(fx.theta_, o_);
+        generate_expression(fx.theta_, user_facing_, o_);
```

Comment:
This could probably use some doc as to why it doesn't match the other ones (it's quoted output, presumably).